scripts/train/train.py

```diff
@@ -454,6 +458,14 @@ def main(cfg: DictConfig) -> Trainer:
         for name, logger_cfg in logger_configs.items()
     ] if logger_configs else None
 
+    if os.environ.get(MOSAICML_PLATFORM_ENV_VAR, 'false').lower(
```

Comment:
nit: Add a comment here that describes this boolean check because the boolean check is quite verbose

Reply:
Alternatively: add a logger.info('...') call that describes this.

Reply:
Just saw daniel's comment from yest: https://github.com/mosaicml/llm-foundry/pull/670#discussion_r1361318147

I think we should also include that check to see if a MosaicMLLogger is already specified as Daniel does in his code comment

Reply:
+1

Reply:
Is it possible to have an instance of `MosaicMLLogger` in `loggers` because `MosaicMLLogger` isn't defined in `build_logger` above ?

Reply:
That is a fair point. I think it is not possible today, but I'd rather include the check for safety against future code.